File: Beginner-Java/1036-uri.java
import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		double A, B, C, Delta, R1, R2;
		
		A = sc.nextDouble();
		B = sc.nextDouble();
		C = sc.nextDouble();
		
		Delta = Math.pow(B, 2.0) - 4.0*A*C;
		
		if (Delta < 0.0 || A == 0) {
			System.out.println("Impossivel calcular");
		}
		else {
			R1 = (-B + Math.sqrt(Delta)) / (2.0 * A);
			R2 = (-B - Math.sqrt(Delta)) / (2.0 * A);
			
			System.out.printf("R1 = %.5f%n", R1);
			System.out.printf("R2 = %.5f%n", R2);
			}
		
		sc.close();
	}
}

File: javascript/Beginner/1036-uri.js
const input = require('fs').readFileSync('stdin', 'utf8');
const [A, B, C] = input.split(' ').map(value => parseFloat(value));

const delta = Math.pow(B, 2) - 4 * A * C;

if (2 * A === 0 || delta < 0) {
   console.log('Impossivel calcular'); 
}

else {
    const R1 = (-B + Math.sqrt(delta)) / (2 * A);
    const R2 = (-B - Math.sqrt(delta)) / (2 * A);

    console.log('R1 = ' + R1.toFixed(5));
    console.log('R2 = ' + R2.toFixed(5));   
}




